### PR TITLE
Add key functionalities documentation

### PR DESCRIPTION
**Pull request by JoelLucaAdams**

### CODE REVIEW

**Comment by ZedThree** [comment]
The new page doesn't have a title, so no link is generated from the table of contents in the index

File: docs/conf.py [hunk 1]
```diff
@@ -29,8 +35,19 @@
...
 ]
 
+# Sometimes the savefig directory doesn't exist and needs to be created
+# https://github.com/ipython/ipython/issues/8733
+# becomes obsolete when we can pin ipython>=5.2; see ci/requirements/doc.yml
+ipython_savefig_dir = os.path.join(
+    os.path.dirname(os.path.abspath(__file__)), "_build", "html", "_static"
+)
+if not os.path.exists(ipython_savefig_dir):
+    os.makedirs(ipython_savefig_dir)
```
Comment (ZedThree) [inline docs/conf.py:49]: Switch to `pathlib.Path`, it makes a lot of this much nicer

File: docs/getting_started.rst [hunk 2]
```diff
@@ -30,40 +30,24 @@ Usage
...
-    print(df["Electric_Field_Ex"])
-    
-    # <xarray.DataArray 'Electric_Field_Ex' (X_x_px_deltaf_electron_beam: 16)> Size: 128B
-    # [16 values with dtype=float64]
-    # Coordinates:
-    #   * X_x_px_deltaf_electron_beam  (X_x_px_deltaf_electron_beam) float64 128B 1...
-    # Attributes:
-    #     units:    V/m
-    #     full_name: "Electric Field/Ex"
+    df = xr.open_dataset("tutorial_dataset_1d/0010.sdf")
```
Comment (ZedThree) [inline docs/getting_started.rst:38]: Should probably use the `with` context manager here so the file gets auto-closed

Comment (JoelLucaAdams) [reply docs/getting_started.rst]: Fixed in https://github.com/PlasmaFAIR/sdf-xarray/pull/34/commits/b404bfb271cd98a7fef88f843c1c89e1717f49fe

**Comment by JoelLucaAdams** [comment]
Alrighty, think I'm finally happy with what I've added. Let me know if you have any changes and if not lets approve and merge

**Pull request merged**